trust-sdk/trust-sdk-specs.md

```diff
@@ -53,48 +89,66 @@ lrzEHpAAIAmoB0Yegc2ZqxtkSkXlYo_TEg4eBrjopGUj9ySxJh6JlfToGqR7yNKzV8cD_yN_jVR5YrVa
 7Tlfb_0eV47yG1Co&id=4
 ```
 
-### Sign Message
+### Sign simple transaction
+
+This command is the simplified version of `sdk_sign`, it accepts a simple `Transaction` object instead of heavier protobuf model
 
 #### Command
-* Command: `sdk_sign_message`
+
+* Command: `sdk_transaction`
 * Parameters:
   * `coin`: BIP44 coin code
```

Comment:
We use assetId: it contains information about coin and tokenId

Reply:
ok, will change this to our UAI